ARCHITECTURE.md

```diff
@@ -0,0 +1,112 @@
+# Standard Design and Architecture
+
+At the time of writing, almost a year of exploratory and freestyle project history has past.
+Fortunately, it is not necessary for further understanding, so I'll spare you that.
+This document, though, lays out the design, architecture and direction of Standard.
+
+If the topics discussed herein are dear to you, please take it as an invitation to get involved.
+
+This design document can only be altered through an RFC process.
+
+## Overview
+
+Standard is a _framework_ to bootstrap and sustain the automatable sections of the Software Delivery Lifecycle (SDLC) _efficiently_ with the power of Nix and Flakes.
+In particular, Standard is a _Horizontal Integration Framework_ which integrates _vertical_ tooling, that is designed around the linux philosophy of doing one thing and to it well.
+The integration target, thereby, are the end-to-end automatable sections of the SDLC process for which we offer well-integrated tools and best practices.
```

Comment:
Question from a relative outsider (sorry if I'm intruding on a developing RFC process):

Could you go into some brief detail about the definitions of "horizontal integration" and "vertical tooling" here? A quick search led me to articles ([example](https://www.investopedia.com/ask/answers/051315/what-difference-between-horizontal-integration-and-vertical-integration.asp)) about business acquisitions, which sort of gives me an idea of the meaning but not so much as it applies to Standard specifically. 

One or two examples per term as they apply to Standard might be helpful (one example `a` would leave it to the reader to identify `n` in the same category, while `a` and `b` would provide more – but not too much – context).

Reply:
That's a good idea.

In short, vertical equals linux philosophy in a narrow scope, whereas horizontal is an overlay that provides the integration across these vertical tools.

Now, I'll have to bounce this a bit in my head to come up with a good word cast of the ideas.

Reply:
Addressed in https://github.com/divnix/std/pull/232/commits/79b840eaa072a2f41fb6b64c0b3457092ac04af8. Does that sound about appropriate?

Reply:
That helps, thanks.